newIDE/app/src/ProjectManager/index.js

```diff
@@ -705,6 +794,8 @@ export default class ProjectManager extends React.Component<Props, State> {
                     }
                     onEdit={() => this.props.onOpenLayout(name)}
                     onDelete={() => this.props.onDeleteLayout(layout)}
+                    onAdd={() => this._onAdd('layout', name, i)}
```

Comment:
Prefer calling directly `this._addLayout` (and same for the rest).
This removes a level of indirection/abstraction which is not really useful and less type safe (here it's not changing anything, but we could imagine that we pass at some point something that is typed like a `gd.Layout` to the function).